packages/e2e/cypress/integration/common/geo.ts

```diff
@@ -16,7 +16,9 @@ When('I clear the search results', () => {
   cy.findByRole('textbox', {
     name: /search/i,
   }).trigger('mouseenter');
-  cy.findByRole('button', { name: 'Clear' }).click();
+  // Adding 'force' as the clear button is hidden until we hover on textbox,
+  // and the click action seems to happen before the hover thus failing the check for the clear button element.
```

Comment:
Last nit, I promise 🙏 :

```suggestion
  /** 
  /**
   * Adding 'force' as the clear button is hidden until we hover on textbox,
   * and the click action happens before the hover.
   */
```

Reply:
Updated